src/ORM/FieldType/DBHTMLVarchar.php

```diff
@@ -124,7 +124,7 @@ public function Plain()
 
     public function scaffoldFormField($title = null, $params = null)
     {
-        return HTMLEditorField::create($this->name, $title, 1);
+        return HTMLEditorField::create($this->name, $title)->setRows(1);
```

Comment:
I'm not sure about changing the default Row count, as this could change the behaviour of projects and force developers to override it. Perhaps we should just remove the default value and leave it at that?

Reply:
```suggestion
        return HTMLEditorField::create($this->name, $title);
```

Reply:
Fixed.